### PR TITLE
Adjust talk list and detail page to 2020 style

### PR DESCRIPTION
**Pull request by yychen**
Also expose the talk list from events menu and hide the cfp menu.

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [x] **Other (page style)**

## Description
**Describe what the change is**
Adapt the 2020 styles for talk list and detail pages. Also expose the page to the menu.

## Steps to Test This Pull Request
1. Go to 'Events' -> 'Talks'
2. Click on any talk link to see the detail page

## Expected behavior
You should see the adapted styles in the Talks (1) and detail page (2).

## Related Issue
If applicable, refernce to the issue related to this pull request.

## More Information
**Screenshots**
圖片

**Additional context**
Thank god the icons are in SVGs so I can change the color without opening any image processing applications :D

### CODE REVIEW

**Comment by yychen** [comment]
I'll make another PR for the community track label.

File: src/templates/pycontw-2020/_includes/menu.html [hunk 1]
```diff
@@ -81,10 +81,17 @@
 		<div class="dropdown">
 			<ul class="submenu">
 				<li><a href="{{ warmup_session }}">{% trans 'Warm-Up Session' %}</a></li>
+				<!--{% comment %}-->
+				<li><a href="{{ events_keynote_url }}">{% trans 'Keynotes' %}</a></li>
+				<!--{% endcomment %}-->
+				<li><a href="{{ events_talk_list_url }}">{% trans 'Talks' %}</a></li>
+				<!--{% comment %}-->
+				<li><a href="{{ events_tutorial_list_url }}">{% trans 'Tutorials' %}</a></li>
+				<li><a href="{{ events_openspaces_url }}">{% trans 'Open Spaces' %}</a></li>
```
Comment (Lee-W) [inline src/templates/pycontw-2020/_includes/menu.html:90]: What will happen if we do not have the content at this moment?

Comment (tai271828) [reply src/templates/pycontw-2020/_includes/menu.html:90]: I expect it won't break anything because we do not have the corresponding contents yet.

Comment (Lee-W) [reply src/templates/pycontw-2020/_includes/menu.html:90]: Nice 🎉 Then, I'll approve this one.

Comment (tai271828) [reply src/templates/pycontw-2020/_includes/menu.html:90]: I double checked the staging site and it does not show anything (as our expectation). I will merge the pull request.

Selection_031

**Pull request merged**